### ISSUE TITLE
Supporting the CLR 4.8

### ISSUE THREAD

**Issue by RealDotNetDave** [dotnet/BenchmarkDotNet#1172]
Does BenchmarkDotNet support the .NET Framework 4.8? I'm unable to get it to work. 

<TargetFrameworks>netcoreapp3.0;net48</TargetFrameworks>

BenchmarkDotNet=v0.11.5, OS=Windows 10.0.17763.503 (1809/October2018Update/Redstone5)
Intel Core i7-7660U CPU 2.50GHz (Kaby Lake), 1 CPU, 4 logical and 2 physical cores
  [Host]     : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.8.3801.0
  Job-KSZIMN : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.8.3801.0

**Comment by billwert** [comment]
Yes, it does. Do you have 4.8 installed on the machine?

> [Host] : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.8.3801.0

**Comment by RealDotNetDave** [comment]
Yes, Have installed it twice.

**Comment by billwert** [comment]
Ah, I got caught up by the TFM. Sorry!

Did you install the targeting pack? https://dotnet.microsoft.com/download/thank-you/net48-developer-pack 

Or you can add a reference to the new nuget package which has the reference assemblies. https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/1.0.0-preview.2

**Comment by adamsitnik** [comment]
Hi @RealDotNetDave 

Windows OS is always going to run all .NET apps using the latest available .NET on your box. In your case, it's 4.8.

The problem is that so far BenchmarkDotNet was always printing not the .NET version number, but .NET **SDK** version number as the first value in the formatted line:

```log
NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.8.3801.0
.NET SDK 4.7.2
.NET v4.8.3801.0
```

This has been fixed in #1114, in the upcoming 0.11.6 BenchmarkDotNet is going to print the right values. In your case it's going to be:

```log
.NET Framework 4.8 (4.8.3801.0), 64bit RyuJIT
```
